spec/system/admin/products_v3/create_spec.rb

```diff
@@ -81,6 +82,10 @@
           find('input[id$="_display_as"]').fill_in with: "2 grams"
           find('button[aria-label="On Hand"]').click
           find('input[id$="_price"]').fill_in with: "11.1"
+
+          select supplier.name, from: 'Producer'
+          select taxon.name, from: 'Category'
```

Comment:
This doesn't actually test the newly added requirements, that the producer and category dropdowns were blank at the beginning. 
That would have been nice, but I think this is still ok.

Reply:
Changes here were to fix failing specs
I agree it will be nice to check when the variant category and producer values are blank